src/Hipace.H

```diff
@@ -182,33 +145,6 @@ public:
      */
     static std::string Version ();
 
-    /** Transverse MPI communicator (for transverse exchanges in 1 slice in the presence of
-     * transverse parallelization)
-     */
-    MPI_Comm m_comm_xy = MPI_COMM_NULL;
```

Comment:
Since MultiBuffer uses a longitudinal communicator within the loop over slices, it would be confusing to push the xy communicator there. It wasn’t used for anything else, so I removed it fully.